README.md

```diff
@@ -54,42 +54,6 @@ $ docker run --name my-running-varnish --tmpfs /usr/local/var/varnish:exec -d -p
 
 Then you can hit `http://localhost:8080` or `http://host-ip:8080` in your browser.
 
-## Advanced configuration using environment variables
```

Comment:
Shouldn't this doc be updated instead of deleted?

Reply:
I think we need to restructure the entire "How to use this image" section anyway. So removing for now...

Reply:
We could follow the structure of https://github.com/docker-library/docs/blob/master/haproxy/README.md#how-to-use-this-image